library/std/src/fs.rs

```diff
@@ -553,7 +553,7 @@ impl File {
     /// the `SetFileInformationByHandle` function on Windows. Note that, this
     /// [may change in the future][changes].
     ///
-    /// [changes]: ../io/index.html#platform-specific-behavior
+    /// [changes]: crate::io#platform-specific-behavior
```

Comment:
Looks like `crate::io` is already imported

```suggestion
    /// [changes]: crate::io#platform-specific-behavior
```